README.md

```diff
@@ -138,6 +138,92 @@ output.render()
 
 ![Fluid tank simulation.](resources/media/fluid_tank.gif)
 
+### Coastal area
+
+This scenario simulates waves propagating in a coastal area represented by a
+bathymetric profile (i.e., the map of depths of the sea bottom as a function
+of spatial coordinates x, y). Waves are injected into the domain from one of the
+boundaries of the simulation with a given amplitude and period. The simulation
+is performed using the [SWASH](https://swash.sourceforge.io/) simulator, which
+solves th [shallow-water equations](https://en.wikipedia.org/wiki/Shallow_water_equations).
+
+#### Example
+
+Start by creating or loading a bathymetry. This can be done e.g. by reading an
+[ASCII XYZ file](https://emodnet.ec.europa.eu/sites/emodnet.ec.europa.eu/files/public/20171127_DTM_exchange_format_specification_v1.6.pdf),
+a standard format for bathymetry data (for examples, see e.g. the [European Marine Observation and Data Network](https://emodnet.ec.europa.eu/geoviewer/#!/)).
+Here we shall use the bathymetry data from Algarve available [here](https://sextant.ifremer.fr/record/SDN_CPRD_590_HR_Lidar_Algarve/)
+as an example.
+
+```python
+from inductiva import fluids
+
+bathymetry = fluids.scenarios.coastal_area.Bathymety.from_ascii_xyz_file(
```

Comment:
I think this is overkill in the main ReadMe, however it is a wonderful introduction to the general methods. You should keep the bathymetry as illustration of the class, but probably with a more simple one where you just load.

So I suggest, we start creating a new branch just for improving the overall documentation. We can start with a ReadME on each scenario folder and then move from there. WDYT?

Reply:
Sure, let's work to improve the organization of the documentation in a future PR.